workflow/full-installertest-flow.groovy

```diff
@@ -0,0 +1,66 @@
+// Full installer test flow, in one file
+// You must parameterize the build with: 
+
+// @stringparameter dockerLabel (node label for docker nodes)
+// @stringparameter packagingBranch - branch to use in packaging build
+
+// **ARTIFACTS URLS**
+// Note: you can use an artifact archived in a Job build by an artifact:// URL
```

Comment:
I'd love it if you can optionally just specify a job/build and it figures out the artifact URLs from that, just to make life a little simpler...personal preference, but given that 99 times out of 100, `debfile`, `rpmfile` and `susefile` are all going to be under the same `artifact://full/path/to/job/buildNr` root (with an assumption that they'll all be directly under there, not in a subdirectory), auto-determination would be pleasant.